src/transformers/models/tapas/configuration_tapas.py

```diff
@@ -129,6 +129,9 @@ class TapasConfig(PretrainedConfig):
             If the aggregation labels are defined and one of these labels represents "No aggregation", this should be
             set to its index. For example, the WTQ models have the "NONE" aggregation label at index 0, so that value
             should be set to 0 for these models.
+        use_cache (:obj:`bool`, `optional`, defaults to :obj:`True`):
+            Whether or not the model should return the last key/values attentions (not used by all models). Only
+            relevant if ``config.is_decoder=True``.
```

Comment:
This shouldn't be added probably, since you can't use TAPAS as a decoder, right?

Reply:
Yes. Removed